components/dashboard/src/components/UsageView.tsx

```diff
@@ -41,9 +41,14 @@ function UsageView({ attributionId }: UsageViewProps) {
         const match = /#(\d{4}-\d{2}-\d{2}):(\d{4}-\d{2}-\d{2})/.exec(location.hash);
         if (match) {
             try {
-                setStartDate(dayjs(match[1], "YYYY-MM-DD"));
-                setEndDate(dayjs(match[2], "YYYY-MM-DD"));
+                const today = new Date();
+                const prevMonth = new Date(today.getFullYear(), today.getMonth() - 1, today.getDate());
+                const todayStr = today.toISOString().slice(0, 10);
+                const prevMonthStr = prevMonth.toISOString().slice(0, 10);
```

Comment:
If I'm reading this correctly, this change would ignore the the URL hash parameter and would always set it to one month back. That's actually not the desired behaviour, as it prevents deep-linking of the current view in the URL.

Showing the last 1 month, could be the default behaviour when no hash is supplied, however, but the change as it stands does not do that. You likely want to move the handling into the `!match` case of this if.